bin/test.sh

```diff
@@ -15,29 +25,13 @@ while true; do
 done
 
 echo
-echo -n "Validating pages "
-
-FILES=$(find src/public -name \*.html | grep -v google)
-PATH=node_modules/.bin:$PATH
-
-exit_code=0
-error_files=""
 
-for file in $FILES; do
-    echo -n .
-    html-validator --quiet --errors-only --validator='http://localhost:8888' --file=$file && continue
-    error_files="$error_files $file"
-    exit_code=1
-done
+docker run --rm --link validator decred/dcrweb-test
 
-echo done
+exit_code=$?
```

Comment:
Adding 'set -e' at the stop will have bash fast fail on any command returning non-zero.

Reply:
In this case though, we want to loop through all of the pages in order to give a report of all problems and THEN fail :)